packages/enzyme/src/selectors.js

```diff
@@ -141,6 +142,10 @@ export function buildPredicate(selector) {
   // If the selector is an non-empty object, treat the keys/values as props
   if (typeof selector === 'object') {
     if (!Array.isArray(selector) && selector !== null && !isEmpty(selector)) {
+      const hasUndefinedValues = values(selector).some(value => value === undefined);
```

Comment:
Why is this one a test of the value itself but the others are testing typeof?

Reply:
My habit is to always use typeof for safety; babel makes this unnecessary. I'll make them all be the same.

Reply:
It's now consistent in the project.